src/main/java/gg/projecteden/nexus/features/chat/alerts/AlertsCommand.java

```diff
@@ -52,16 +52,26 @@ void list() {
 			JsonBuilder builder = new JsonBuilder();
 
 			if (highlight.isPartialMatching()) {
-				builder.next("&a✔")
-						.command("/alerts partialmatch false " + highlight.getHighlight())
-						.hover("&cClick to turn off partial matching");
+				builder.group().next("&d⦿ ")
```

Comment:
I believe calling group() should come after you setup the Component, so .next().command().hover().group(), Just to be consistent with the rest of the code base, add it after the last builder method used, not on the next builder call.